pkg/portal/config/admin_api.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import (
+	"github.com/authgear/authgear-server/pkg/lib/config"
+)
+
+type AdminAPIType string
+
+const (
+	AdminAPITypeLocal AdminAPIType = "local"
```

Comment:
I suppose this name is misleading? In this mode, the admin API endpoint is configured by providing a **static** endpoint, and the endpoint may not be a *local* endpoint. I propose changing it to `static` instead.